src/GCodePathConfig.h

```diff
@@ -29,9 +29,10 @@ class GCodePathConfig
     const int line_width; //!< width of the line extruded
     const int layer_thickness; //!< current layer height in micron
     const double flow; //!< extrusion flow modifier in %
+    const double extrusion_speed_factor; //! < extrusion speed factor (only used by purge moves for now)
```

Comment:
This is actually a travel speed factor, but this is only for the travel speed during extrusion, so pure travel moves should not be affected by this. That's why I named it as `extrusion_speed_factor`, but then this can still be confused with the speed of the extruder motor(s).